lib/init/config-initializer.js

```diff
@@ -445,114 +450,121 @@ function askInstallModules(modules, packageJsonExists) {
     });
 }
 
-/* istanbul ignore next: no need to test inquirer */
+/* istanbul ignore next: no need to test enquirer */
 /**
  * Ask use a few questions on command prompt
  * @returns {Promise} The promise with the result of the prompt
  */
 function promptUser() {
 
-    return inquirer.prompt([
+    return enquirer.prompt([
         {
-            type: "list",
+            type: "select",
             name: "purpose",
             message: "How would you like to use ESLint?",
-            default: "problems",
+            initial: 1,
             choices: [
-                { name: "To check syntax only", value: "syntax" },
-                { name: "To check syntax and find problems", value: "problems" },
-                { name: "To check syntax, find problems, and enforce code style", value: "style" }
+                { message: "To check syntax only", name: "syntax" },
+                { message: "To check syntax and find problems", name: "problems" },
+                { message: "To check syntax, find problems, and enforce code style", name: "style" }
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "moduleType",
             message: "What type of modules does your project use?",
-            default: "esm",
+            initial: 0,
             choices: [
-                { name: "JavaScript modules (import/export)", value: "esm" },
-                { name: "CommonJS (require/exports)", value: "commonjs" },
-                { name: "None of these", value: "none" }
+                { message: "JavaScript modules (import/export)", name: "esm" },
+                { message: "CommonJS (require/exports)", name: "commonjs" },
+                { message: "None of these", name: "none" }
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "framework",
             message: "Which framework does your project use?",
-            default: "react",
+            initial: 0,
             choices: [
-                { name: "React", value: "react" },
-                { name: "Vue.js", value: "vue" },
-                { name: "None of these", value: "none" }
+                { message: "React", name: "react" },
+                { message: "Vue.js", name: "vue" },
+                { message: "None of these", name: "none" }
             ]
         },
         {
-            type: "confirm",
+            type: "toggle",
             name: "typescript",
             message: "Does your project use TypeScript?",
-            default: false
+            enabled: "Yes",
+            disabled: "No",
+            initial: 0
         },
         {
-            type: "checkbox",
+            type: "multiselect",
```

Comment:
Does the `enquirer` support a help message for `multiselect`?
IMO, it would be nice if we give help message for it

* inquirer (master branch version)
![스크린샷 2020-05-05 오후 10 48 50](https://user-images.githubusercontent.com/41323220/81074197-9e426480-8f23-11ea-80a5-35b1f306e4dd.png)

* enquirer (pr version)
![스크린샷 2020-05-05 오후 10 47 56](https://user-images.githubusercontent.com/41323220/81074230-abf7ea00-8f23-11ea-8548-adb419d3f4c0.png)

Reply:
Not really, but I've added it to the message to clear the usage up.